src/main/java/com/regnosys/testing/testpack/TestPackFunctionRunnerImpl.java

```diff
@@ -74,8 +75,9 @@ public TestPackFunctionRunnerImpl(Function<IN, RosettaModelObject> function,
     }
 
     @Override
-    public Pair<String, Assertions> run(Path inputPath) {
-        URL inputFileUrl = Resources.getResource(inputPath.toString());
+    public Pair<String, Assertions> run(Path inputPath) throws MalformedURLException {
+        inputPath = ROSETTA_SOURCE_PATH.resolve(inputPath);
```

Comment:
Assign to new variable `inputPathFromRepositoryRoot` (or something) rather than reassign